### PR TITLE
knative: update values-metadata descriptions and image security readme

### PR DESCRIPTION
**Pull request by tomleah**

### CODE REVIEW

File: community/knative/README.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ This chart installs Knative components for Build, Serving, Eventing and Eventing
 
 ## Chart Details
 
-This chart is comprised of multiple subcharts which is illustrated in the structure below:
+This chart is comprised of multiple subcharts which is illustrated in the structure below. 
+Disabling a chart will disable all charts below it in the chart structure. When enabling a subset of charts note that the parent charts are prerequisites and must be installed previously or in conjunction.
 ```
 knative                         (default)
```
Comment (mhamwala) [inline community/knative/README.md]: we had a discussion about removing the default tags?

**Review by mhamwala — CHANGES_REQUESTED** [review]
LGTM after changes

File: community/knative/README.md (hunk 1 above)
Comment (tomleah) [reply community/knative/README.md]: Added (requires: X enabled)

File: community/knative/values-metadata.yaml [hunk 2]
```diff
@@ -448,17 +436,15 @@ serving:
...
-        description: Toggle to enable/disable Knative Monitoring
+        label: "Enable Knative Monitoring (requires: serving enabled)" 
+        description: Toggle to enable/disable Knative Monitoring. By default Prometheus, In-Memory Zipkin and Elasticsearch is installed. See further Monitoring configuration to enable/disable monitoring components.
         type: boolean
-        required: true
     prometheus:
       enabled:
         __metadata:
-          label: Enable Prometheus
+          label: "Enable Prometheus (requires: monitoring)"
```
Comment (mhamwala) [inline community/knative/values-metadata.yaml]: add enabled*

**Review by mhamwala — DISMISSED** [review]
LGTM.
address nits if you wish

File: community/knative/values-metadata.yaml [hunk 3]
```diff
@@ -414,10 +403,9 @@ serving:
             type: number
       enabled:
         __metadata: 
-          label: Enable Elasticsearch
+          label: "Enable Elasticsearch (requires: monitoring)"
```
Comment (mhamwala) [inline community/knative/values-metadata.yaml]: add enabled

**Review by mhamwala — APPROVED** [review]
LGTM

**Pull request merged**